File: ADT102/AnotherThirdAssignment/Student.py
class Student:
    st_id = ''
    title = ''
    name = ''
    last_name = ''

    def __init__(self, st_id, title, name, last_name):
        self.st_id = st_id
        self.title = title
        self.name = name
        self.last_name = last_name

    def __str__(self):
        return "id: ", self.st_id, " title: ", self.title, " name ", self.name, " last name: ", self.last_name

File: ADT102/queue&stack/Student.java
package app;

public class Student {
    private String ID;
    private String Title;
    private String Name;
    private String Surname;

    public Student(String ID, String title, String name, String surname) {
        this.ID = ID;
        this.Title = title;
        this.Name = name;
        this.Surname = surname;
    }

    @Override
    public String toString() {
        return "Student{" +
                "ID='" + ID + '\'' +
                ", Title='" + Title + '\'' +
                ", Name='" + Name + '\'' +
                ", Surname='" + Surname + '\'' +
                '}';
    }
}
